app_template/manifest.json.tt

```diff
@@ -7,7 +7,9 @@
   },
   "defaultLocale": "en",
   "private": true,
-  "location": "ticket_sidebar",
+  "location": { "zendesk":
+    { "ticket_sidebar": "<%= @iframe_location %>" }
```

Comment:
As in you want them to put `_legacy` in their manifest? I thought we were just going to stick with string/arrays. I'm fine with it if the rest of the team are onboard...